src/typeDefs/schema.graphql

```diff
@@ -92,6 +92,22 @@ type SpokenLanguage {
   nameNative: String!
 }
 
+type Submission {
+  id: ID!
+  submittedLink: String!
```

Comment:
what kind of link? This should be something like `googleMapsUrl`

Reply:
Yes, that's definitely more descriptive. I'll make the change as soon as I have the chance.